backend/flask_main.py

```diff
@@ -1,26 +1,20 @@
 import json
 from datetime import timedelta
+from typing import Optional
 
 from flask import Flask, abort, request, session
 from flask_cors import CORS
 from flask_login import LoginManager, current_user, login_user, logout_user
 from flask_redis import FlaskRedis
 
 from .exceptions import RedisKeyNotFoundError
+from .local_settings import FLASK_SECRET_KEY, REDIS_DB, REDIS_PASSWORD, REDIS_PORT, REDIS_URL
 from .query_tools import get_user_by_email
+from .redis_schema import *
+from .scrum_types import FLASK_RESPONSE_TYPE
 from .structs import HTTP_STATUS_CODE, AnonymousUserWrapper, WebsocketInfo
 from .utils import cleanup_redis_dict
 
-try:
-    from .local_settings import SERVER_NAME, REDIS_URL, REDIS_PASSWORD, REDIS_PORT, REDIS_DB, FLASK_SECRET_KEY
```

Comment:
Before I had mike put creating a local settings file into the repo, this would have happened for everyone who didn't know to make one.  I didnt want to break people's apps but it was intended to be a temp hack